hub/apps/get-started/uno-simple-photo-viewer.md

```diff
@@ -0,0 +1,622 @@
+
+        // Create a bitmap to be the image source.
+        BitmapImage bitmapImage = new();
+        bitmapImage.SetSource(fileStream);
+
+        return bitmapImage;
+    }
+
+    public async Task<BitmapImage> GetImageThumbnailAsync()
+    {
+        StorageItemThumbnail thumbnail =
+            await ImageFile.GetThumbnailAsync(ThumbnailMode.PicturesView);
+        // Create a bitmap to be the image source.
+        var bitmapImage = new BitmapImage();
+        bitmapImage.SetSource(thumbnail);
+        thumbnail.Dispose();
+
+        return bitmapImage;
+    }
+
+    public string ImageName { get; }
+
+    public string ImageFileType { get; }
+
+    public string ImageDimensions => $"{ImageProperties.Width} x {ImageProperties.Height}";
+
+    public string ImageTitle
+    {
+        get => string.IsNullOrEmpty(ImageProperties.Title) ? ImageName : ImageProperties.Title;
+        set
+        {
+            if (ImageProperties.Title != value)
+            {
+                ImageProperties.Title = value;
+                _ = ImageProperties.SavePropertiesAsync();
+                OnPropertyChanged();
+            }
+        }
+    }
+
+    public int ImageRating
+    {
+        get => (int)ImageProperties.Rating;
+        set
+        {
+            if (ImageProperties.Rating != value)
+            {
+                ImageProperties.Rating = (uint)value;
+                _ = ImageProperties.SavePropertiesAsync();
+                OnPropertyChanged();
+            }
+        }
+    }
+
+    public event PropertyChangedEventHandler PropertyChanged;
+
+    protected void OnPropertyChanged([CallerMemberName] string propertyName = null) =>
+        PropertyChanged?.Invoke(this, new PropertyChangedEventArgs(propertyName));
+}
+```
+
+This class will serve as a model to represent the image files in the `GridView`. In the next sections, we will make a set of changes to these copied files to make them compatible in a multi-platform context.
+
+### Add image assets to the project
+
+In the `UnoSimplePhotos` project, create a new folder named `Assets` and copy the JPG image files to a `Samples` subfolder. The `Assets` folder structure should now look like this:
+
+[Insert image here]
+
+### Making the code multi-platform capable
+
+In the desktop project from the previous tutorial, the `MainPage.xaml.cs` file contains a `GetItemsAsync` method that enumerates items from a `StorageFolder` representing the installed package location. Because that location is not available on certain platforms such as WebAssembly, we will need to make changes to this method to make it compatible with all platforms. We will accordingly make some changes to the `ImageFileInfo` class to ensure compatibility.
+
+First, we will make changes to the `GetItemsAsync` method. Replace the `GetItemsAsync` method in the `MainPage.xaml.cs` file with the following code:
+
+```csharp
+private async void GetItems()
```

Comment:
This should be GetItemsAsync and we should return Task so that it can be awaited